Add `Zip` for multiple Publishers (#6)
CombineCommunity/CombineExt: README.md

odd: 5
```

------

### ZipMany

> `ZipMany.swift` contains two overloads on Combine’s `Publisher.zip` methods (which, at the time of writing only go up to arity three).

Reviewer: Uhm sorryp 
```suggestion
This repo includes two overloads on Combine’s `Publisher.zip` methods (which, at the time of writing only go up to arity three).
```

> This allows folks to zip arbitrarily many publishers and receive either an array of inner publisher outputs back.

Reviewer: ```suggestion
This lets you arbitrarily zip many publishers and receive either an array of inner publisher outputs back.
```
